README.md

```diff
@@ -4,15 +4,15 @@
 [![Pub](https://img.shields.io/pub/v/optimizely_flutter_sdk.svg)](https://pub.dev/packages/optimizely_flutter_sdk)
 [![Coverage Status](https://coveralls.io/repos/github/optimizely/optimizely-flutter-sdk/badge.svg?branch=master)](https://coveralls.io/github/optimizely/optimizely-flutter-sdk?branch=master)
```

Comment:
The coverage status is unknown, can you fix that OR remove the badge and create a story for fixing and re-adding the badge?

Reply:
The GitHub build status is failing. Can you fix that OR remove the badge and create a story for fixing and re-adding the badge?

Reply:
Created FSSDK-8990

Reply:
W/Griffin away rn, I can weight in. Let's Remove both for now and add them back when they're both fixed.